### DISCUSSION TITLE
求缺氧 R2版

### DISCUSSION THREAD

**Discussion by luqiyihui** [Gamuxorg/bbs#334] [投递和请求游戏]
求缺氧R2版，这样算催更新贴吗？实在不好意思，但比较喜欢这个游戏

**Comment by shouhuanxiaoji** [comment]
你好，已经更新，今晚放出下载链接

**Comment by luqiyihui** [comment]
版主，你好，请问缺氧更新了吗？链接好像是之前的版本号啊

**Comment by shouhuanxiaoji** [comment]
好了

**Comment by luqiyihui** [comment]
版主，你好，游戏没有成功运行，在终端下显示：
[luqi@Archlinux Downloads]$ ./oxygennotincluded_cu-273433_amd64.appimage 
Found path: /tmp/.mount_oxygenjHWlsj/usr/bin/OxygenNotIncluded.x86_64
Mono path[0] = '/tmp/.mount_oxygenjHWlsj/usr/bin/OxygenNotIncluded_Data/Managed'
Mono path[1] = '/tmp/.mount_oxygenjHWlsj/usr/bin/OxygenNotIncluded_Data/Mono'
Mono config path = '/tmp/.mount_oxygenjHWlsj/usr/bin/OxygenNotIncluded_Data/Mono/etc'
Preloaded 'libCSteamworks.so'
Preloaded 'libSimDLL.so'
Preloaded 'libfmod.so'
Preloaded 'libfmodL.so'
Preloaded 'libfmodstudio.so'
Preloaded 'libfmodstudioL.so'
Preloaded 'libsdkencryptedappticket.so'
Preloaded 'libsteam_api.so'
Player data archive not found at `/tmp/.mount_oxygenjHWlsj/usr/bin/OxygenNotIncluded_Data/data.unity3d`, using local filesystem
[luqi@Archlinux Downloads]$ 

请问我该怎么调整？

**Comment by shouhuanxiaoji** [comment]
详细说一下系统信息
另外解压一下，./xxx.appimage --appimage-extract
解压后也可以运行，看看有没有报错

**Comment by luqiyihui** [comment]
系统是Archlinux
解压后运行结果如下：
[luqi@Archlinux squashfs-root]$ ./AppRun 
Found path: /home/luqi/Downloads/squashfs-root/usr/bin/OxygenNotIncluded.x86_64
Mono path[0] = '/home/luqi/Downloads/squashfs-root/usr/bin/OxygenNotIncluded_Data/Managed'
Mono path[1] = '/home/luqi/Downloads/squashfs-root/usr/bin/OxygenNotIncluded_Data/Mono'
Mono config path = '/home/luqi/Downloads/squashfs-root/usr/bin/OxygenNotIncluded_Data/Mono/etc'
Preloaded 'libCSteamworks.so'
Preloaded 'libSimDLL.so'
Preloaded 'libfmod.so'
Preloaded 'libfmodL.so'
Preloaded 'libfmodstudio.so'
Preloaded 'libfmodstudioL.so'
Preloaded 'libsdkencryptedappticket.so'
Preloaded 'libsteam_api.so'
Player data archive not found at `/home/luqi/Downloads/squashfs-root/usr/bin/OxygenNotIncluded_Data/data.unity3d`, using local filesystem[luqi@Archlinux squashfs-root]$ 

在解压文件夹中也没有看到/home/luqi/Downloads/squashfs-root/usr/bin/OxygenNotIncluded_Data/data.unity3d文件

下面怎么办？

**Comment by shouhuanxiaoji** [comment]
我看一下怎么解决。晚上回复您

**Comment by luqiyihui** [comment]
好的，给你添麻烦了，谢谢！

**Comment by luqiyihui** [comment]
弱弱的问下，可有进展？

**Comment by shouhuanxiaoji** [comment]
magnet:?xt=urn:btih:D28CAAB72487FA4B31BFED9E0E59723A03C6FBD1&tr=http%3A%2F%2Fbt.t-ru.org%2Fann%3Fmagnet&dn=%5B%D0%92%20%D1%80%D0%B0%D0%B7%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D0%BA%D0%B5%5D%20Oxygen%20Not%20Included%20(CU-273433)%20%5Bamd64%5D%20%5BMulti%5D%20%5BUnity3D%5D
试一下这个

**Comment by luqiyihui** [comment]
这个没有问题，运行正常，谢谢！
